man/docker-build.1.md

```diff
@@ -134,22 +134,20 @@ option can be set multiple times.
    about valid tag names.
 
 **-m**, **--memory**=*MEMORY*
-  Memory limit
+   Memory limit
 
-**--memory-swap**=*LIMIT*
-   A limit value equal to memory plus swap. Must be used with the  **-m**
-(**--memory**) flag. The swap `LIMIT` should always be larger than **-m**
-(**--memory**) value.
+**--memory-swap**=*number*[*S*]
+   Combined memory plus swap limit; *S* is an optional suffix which can be one of **b**, **k**, **m**, or **g**.
```

Comment:
The old version described what these units stood for (kilobytes, megabytes etc); could you add back that information?